.github/workflows/ci.yml

```diff
@@ -41,7 +42,7 @@ jobs:
           working-directory: packages/test_track_test_support
 
       - name: Run tests and generate coverage report
-        run: melos exec --fail-fast -- "../../tool/generate_code_coverage.sh"
+        run: melos exec -c 1 --fail-fast -- "../../tool/generate_code_coverage.sh"
```

Comment:
Needed to restrict concurrency here, otherwise we were running into all sorts of file system issues. Likely if we pulled the package activation into the yaml file and ran the scripting afterwards, that would've worked. But this feels a bit cleaner since we allow the scripting to activate what it needs (and I'm not overly concerned with speed right now).